ui/scss/init/_gui.scss

```diff
@@ -2061,28 +2061,6 @@ input:not(input:-webkit-autofill)::-webkit-contacts-auto-fill-button {
     margin-top: 10px;
   }
 
-  .banner_logo {
-    display: block;
-    float: left;
-    margin-top: 20px;
-    margin-left: 80px;
-    margin-right: 30px;
-    width: 150px;
-    height: 150px;
-    background-image: url('/public/img/cookie.svg');
-    background-size: contain;
-    background-repeat: no-repeat;
-
-    @media (max-width: $breakpoint-medium) {
-      margin-left: 30px;
-      margin-right: 10px;
-    }
-
-    @media (max-width: $breakpoint-small) {
-      display: none;
-    }
-  }
-
```

Comment:
This is still needed and we are still serving the image file.
Is the flagged error something to do with the svg?

Reply:
In this case, webpack couldn't find the file.  I was assuming it was something off with a copy or something, but then I went looking for the class reference and couldn't find anything. 

Additionally, I couldn't find the file in production. Verifying that `public` wasn't nested under another directory with this https://odysee.com/public/img/placeholderTx.gif, looking for `cookie.svg` in that directory turned up nothing: https://odysee.com/public/img/cookie.svg

Given this, I didn't look further since I was thinking the class was dead code.  If this should still be present, it may be highlighting an existing issue in production.

Reply:
> _If this should still be present, it may be highlighting an existing issue in production._

You're right -- it's missing an import trigger.  Someone else will be fixing this in production now.
Thanks for finding this :+1: 

_I'll delete the other thread since it is about the same thing_

Reply:
A fix that aimed to served back the image was pushed to `master`. 

<img width="234" alt="image" src="https://user-images.githubusercontent.com/64950861/181269056-97c6c780-e3ca-4607-8b50-63e9c37e68fe.png">

Let's see if webpack still complains after you rebase.

I think it will, and if it does, the fix would probably to change
  `background-image: url('/public/img/cookie.svg');`
to
  `background-image: url('~static/img/cookie.svg');`
so that `file-loader` can grab the image for us, instead of manually copying it over in b5fa2d68

Reply:
Yup. This did it. I'll put this and the announcement bit back in. I'll push this rebase up after I make a couple more tweaks.

Also, a slight aside. While looking at this, i noticed that the css is currently rolled into the js. Is this what's ultimately intended at this point, or is this a vestige of its previous life as an electron app?

I did a quick test with the extract plugin and the entry js file fell from 6.1MB to like 3.9, which I'd imagine would significantly reduce download time and and probably parse even more so for mobile.

Reply:
Actually, I see you've already got that covered in this ticket: https://github.com/OdyseeTeam/odysee-frontend/issues/1576

Reply:
Yeah, hope that one gets some action soon.  
It's unofficially 'assigned' to someone, but if you're familiar and interested (and assuming the person doesn't mind), maybe can collaborate.